clang/test/CodeGen/ubsan-src-ignorelist-category.test

```diff
@@ -0,0 +1,43 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -fsanitize=signed-integer-overflow -fsanitize-ignorelist=%t/src.ignorelist -emit-llvm %t/test1.c -o -             | FileCheck %s --check-prefixes=CHECK1,IGNORE
```

Comment:
I assume when you land a fix, you just need to update --check-prefixes=
and remove FIXME